### PR TITLE
v0.6.0 release

### PR DESCRIPTION
**Pull request by rebelinux**
## Description

[0.6.0] - 2021-12-02
- Added
    - Added Vserver CIFS Client Session information.
    - Added Storage Aggregate Option Information.

- Changed
    - The network section has been changed to show the content per node.
    - Updated HTML Sample Report.
    - Implemented the ability to specify the InfoLevel option.

- Fixed
    - Fix Volume SnapShot Section logic to display content only when there are snapshots data available.

## Related Issue

- None

## Motivation and Context

- Main change is related to implement aditional InfoLevel options.

## How Has This Been Tested?

- Tested on my homelab

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,12 +148,13 @@ The **Options** schema allows certain options within the report to be toggled on
 
 The **InfoLevel** schema allows configuration of each section of the report at a granular level. The following sections can be set.
 
-There are 2 levels (0-1) of detail granularity for each section as follows;
+There are 4 levels (0-3) of detail granularity for each section as follows;
```
Comment (tpcarman) [inline README.md:151]: There are only 3 InfoLevels, (0-2)

File: README.md [hunk 2]
```diff
@@ -148,12 +148,13 @@ The **Options** schema allows certain options within the report to be toggled on
...
 
 | Setting | InfoLevel         | Description                                                                                                                                |
 |:-------:|-------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
 |    0    | Disabled          | Does not collect or display any information                                                                                                |
 |    1    | Enabled | Provides condensed, detailed information for a collection of objects                                                                                 |
+|    2    | Adv Summary       | Provides condensed, detailed information for a collection of objects                                                                       |
```
Comment (tpcarman) [inline README.md:157]: InfoLevel 2 description is exactly the same as InfoLevel 1,

Here's how i normally do it 

| Setting | InfoLevel         | Description                                                                                                                                |
|:-------:|-------------------|--------------------------------------------------------------------------------------------------------------------------------------------|
|    0    | Disabled          | Does not collect or display any information                                                                                                |
|    1    | Enabled / Summary | Provides summarised information for a collection of objects                                                                                |
|    2    | Adv Summary       | Provides condensed, detailed information for a collection of objects

File: AsBuiltReport.NetApp.ONTAP.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
 
     },
     "InfoLevel": {
-        "_comment_": "0 = Disabled, 1 = Enabled",
+        "_comment_": "0 = Disabled, 1 = Enabled, 2 = Adv Summary",
```
Comment (tpcarman) [inline AsBuiltReport.NetApp.ONTAP.json:15]: I normally do `1 = Enabled / Summary`

**Review by tpcarman — CHANGES_REQUESTED** [review]
I don't see an updated module manifest. Have you increased the version number within the manifest?

File: AsBuiltReport.NetApp.ONTAP.json (hunk 3 above)
Comment (rebelinux) [reply AsBuiltReport.NetApp.ONTAP.json:15]: Done!

File: README.md (hunk 2 above)
Comment (rebelinux) [reply README.md:157]: Done!

File: README.md (hunk 1 above)
Comment (rebelinux) [reply README.md]: Done!

**Pull request merged**